backend/catalogue/management/commands/generate_survey_graphs.py

```diff
@@ -0,0 +1,172 @@
+from django.core.management.base import BaseCommand
+from django.core.files.storage import default_storage
+from django.core.files import File
+from django.db import connections
+from collections import namedtuple
+import matplotlib.pyplot as plt
+from matplotlib.ticker import MaxNLocator
+from io import BytesIO
```

Comment:
Probably a useful habit to get into by including logging (a chat I've been having with Oli lately)

Reply:
Done!